app/src/test/kotlin/compiler/utils/TestUtils.kt

```diff
@@ -0,0 +1,14 @@
+package compiler.utils
+
+import kotlin.test.assertContains
+
+fun <E> assertContentEquals(expected: Set<E>, actual: Set<E>) {
+    for (element in expected)
+        assertContains(actual, element)
+    for (element in actual)
+        assertContains(expected, element)
+}
+
```

Comment:
According to the [docs](https://kotlinlang.org/api/latest/kotlin.test/kotlin.test/assert-content-equals.html), normal `assertEquals` should do exactly this. Or am I missing something? :thinking:

Reply:
That is right. I though that `assertEquals` also checks if the types are the same. We didn't actually need Kotlin's `assertContentEquals` at all, so I replaced all its uses with `assertEquals`.